src/Sylius/Bundle/ResourceBundle/Controller/ResourcesCollectionProvider.php

```diff
@@ -61,9 +61,12 @@ public function get(RequestConfiguration $requestConfiguration, RepositoryInterf
 
         if ($paginator instanceof Pagerfanta) {
             $request = $requestConfiguration->getRequest();
+            if (null !== $limit = $request->query->get('limit')) {
```

Comment:
How about making it a one-liner `$limit = $request->query->has('limit') ? $request->query->getInt('limit') : null`? A little bit more readable for me (no assignment in a condition).

Reply:
done